model/jpa/src/main/java/org/keycloak/models/jpa/JpaClientProviderFactory.java

```diff
@@ -36,19 +37,22 @@
 
     private Set<String> clientSearchableAttributes = null;
 
+    private static final List<String> REQUIRED_SEARCHABLE_ATTRIBUTES = Arrays.asList(
+      "saml_idp_initiated_sso_url_name"
```

Comment:
Couldn't we use the `SamlProtocol.SAML_IDP_INITIATED_SSO_URL_NAME` constant here?

Reply:
Sadly, no, this is defined in services and model-jpa does not and should not depend on services

Reply:
I was suspecting something like that...